jep/7/README.adoc

```diff
@@ -67,22 +69,68 @@ endif::[]
 
 == Abstract
 
-Due to its negative impact on core maintenance, distribution of the unmaintained `ruby-runtime` plugin will be suspended.
-Distribution of plugins depending on it will be suspended until their dependency is removed.
+Due to its negative impact on core maintenance, distribution of the unmaintained
+link:https://github.com/jenkinsci/ruby-runtime-plugin[Ruby Runtime plugin] and
+link:https://github.com/jenkinsci/jenkins.py[Jenkins Python SDK] will be suspended.
```

Comment:
There is no technical reason for us to suspend `python-wrapper`. No workarounds or hacks support it. So let's not pretend there is a technical reason for doing that.

Reply:
I believe there is a consensus in the dev list we want to deprecate it as is while we are there. Do you disagree with that and suggest reverting the Python wrapper change entirely? Or is it just a wording concern?

Reply:
The point is that it is simply untrue that we deprecate it _due to its negative impact on core maintenance_.